templates/workloads/partials/cf/addons.yml

```diff
@@ -1,6 +1,7 @@
 AddonsStack:
   # An Addons CloudFormation Stack for your additional AWS resources
   Type: AWS::CloudFormation::Stack
+  DependsOn: EnvControllerAction
```

Comment:
Will this work for Backend services?

Reply:
oh right i need to skip backend services and jobs.

Reply:
This works pretty well, but we'll have to remember to update this conditional if we add the envcontroller to another pattern.   
I wonder if there is a way of checking the `EnvControllerAction` exists in the template, maybe this? 🤔 
```go
{{- if ne .EnvControllerLambda "" }} # Means the lambda code is injected
  DependsOn: EnvControllerAction
{{- end}}
```

Reply:
Thought about it. However, I was thinking in the future we might depend on the env controller to manage our cross service resources, which means env controller should not be unique for load balanced service. Plus using env controller as condition is not very straight forward. What do you think?

Reply:
Yeah, the EnvController should definitely not be a unique resource for the LBWebService, maybe we can add a method to `WorkloadOpts`:
```go
func (o *WorkloadOpts) HasEnvControllerEnabled() bool {
    return o.EnvControllerLambda != ""
}
```
so that the template reads:
```go
{{- if . HasEnvControllerEnabled }}
  DependsOn: EnvControllerAction
{{- end}}
```
I still think this is more robust then passing the type of the workload where we can forget to update this condition.

Reply:
ok done!